content/en/docs/Getting started/deploying-cuebot.md

```diff
@@ -20,6 +20,10 @@ management tasks, including:
 A typical OpenCue deployment runs a single instance of Cuebot, which is shared
 by all users.
 
+Cuebot functions best when it has a low-latency connection to the
```

Comment:
Wondering if this fits more naturally under 'System requirements' directly below?

Also suggest rewording slightly:

Cuebot functions best when it has a low-latency connection to the
[OpenCue database](/docs/getting-started/setting-up-the-database). We recommend that
you run Cuebot, either on the same machine or within the same local network as the database.